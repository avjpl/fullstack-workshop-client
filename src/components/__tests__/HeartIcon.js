import React from 'react';
import HeartIcon from '../HeartIcon';
import renderer from 'react-test-renderer';

describe('Heart Icon', () => {
  it('renders', () => {
    const comp = renderer.create(<HeartIcon />);
    expect(comp.toJSON()).toMatchSnapshot();
  });
});
