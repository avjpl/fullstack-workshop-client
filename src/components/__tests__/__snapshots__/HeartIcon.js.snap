// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heart Icon renders 1`] = `
<svg
  height="40"
  onClick={undefined}
  viewBox="0 0 30 30"
  width="40"
>
  <path
    d="M23.48 7.9a5.1 5.1 0 0 0-7.39 0l-1 1-1-1a5.1 5.1 0 0 0-7.39 0 5.53 5.53 0 0 0 0 7.64l1 1 7.39 7.64 7.39-7.64 1-1a5.53 5.53 0 0 0 0-7.64z"
    fill="transparent"
    stroke="red"
    strokeWidth={1.5}
  />
</svg>
`;
